src/main/java/org/myrobotlab/document/transformer/XPathExtractor.java

```diff
@@ -172,17 +172,15 @@ private void processXml(String xml, Document doc) throws SAXException, IOExcepti
         }
       }
     } catch (IOException e) {
-      System.out.println("IO Exception reading from file " + filename);
-      e.printStackTrace();
+      log.warn("IO Exception reading from file {} : {}" , filename, e);
       // return what we can...

```

Comment:
this way only gets e.toString() or e.getMessage()  ... if you don't put the auto formatting bracket you get the whole stack trace .. e.g.

log.warn("IO Exception reading from file {} : " , filename, e);

Reply:
doh!